Makefile

```diff
@@ -695,6 +664,7 @@ central-db-image: $(CURDIR)/image/postgres/bundle.tar.gz $(CURDIR)/image/postgre
 .PHONY: clean
 clean: clean-image
 	@echo "+ $@"
+	docker volume rm $(GOPATH_VOLUME_NAME) $(GOCACHE_VOLUME_NAME) || true
```

Comment:
Should we really remove the `/go` and `/gocache` folders on a `clean` invocation?
With the gocache, we shouldn't break anything, except that on the next build all deps would be redownloaded.
But could removing the /go folder not potentially lead to loss of work? AFAIK we recommend checking out `stackrox/stackrox`, `stackrox/scanner`, and `stackrox/workflows` to `/go`, don't we? Wouldn't all these with all the local branches people potentially have be deleted on a `clean`?

Reply:
Thank you for the clarification. 
I remembered that bind mounts are mounting directly from the host filesystem, so I wanted to make sure that a `docker rm BIND_MOUNT_XYZ` doesn't clear out the folder on the host.